src/domain/entities/equipamentEnum/status.ts

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable no-unused-vars */
 export enum Status {
-  ACTIVE = 'ACTIVE',
-  ACTIVE_BY_DEMISE = 'ACTIVE_BY_DEMISE',
-  INACTIVE = 'INACTIVE',
-  MAINTENANCE = 'MAINTENANCE',
-  DOWNGRADED = 'DOWNGRADED',
-  TECHNICAL_RESERVE = 'TECHNICAL_RESERVE'
+  ACTIVE = 'Ativo',
+  ACTIVE_LOAN = 'Ativo Empréstimo',
+  DOWNGRADED = 'Baixo',
```

Comment:
Só por uma questão do que ficaria mais próximo da fala dos clientes... será que "baixado" seria melhor que "baixo" ?

```suggestion
  DOWNGRADED = 'Baixado',
```

Reply:
Houve o erro no termo mesmo, o correto era "baixado". Fiz a alteração.